migrator/migrations/m_202_to_m_203_vuln_requests_for_suppressed_cves/migration_test.go

```diff
@@ -96,8 +96,10 @@ func (s *migrationTestSuite) TestMigration() {
 	}
 
 	now := protocompat.TimestampNow()
-	expiry := now.Clone()
-	expiry.Seconds += int64(7 * 24 * time.Hour.Seconds())
+	expiry := &protocompat.Timestamp{
+		Seconds: now.Seconds + int64(7*24*time.Hour.Seconds()),
+		Nanos:   now.Nanos,
```

Comment:
Do we need the Nanos?

Reply:
To be honest, I don't know. That is the implementation detail that I'm not aware of. If we look at the previous implementation, we had Nanos, and this change preserves that behavior.

Reply:
Fair enough.  Not terribly important in this case.  I was just curious.